ten_million_with_a_hat.module

```diff
@@ -106,8 +106,14 @@ function ten_million_with_a_hat_ingest_batch_operation($objects, $user, $cm_set,
     // Let's make an object wheeeee.
     $tuque = islandora_get_tuque_connection();
     $object = $tuque->repository->constructObject($namespace);
-    $completed = $sandbox['completed'] + 1;
-    $object->label = "Hat #" . $completed;
+    $characters = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ ';
+    $character_length = strlen($characters);
+    $random_string = '';
+    $length = 10;
+    for ($i = 0; $i < $length; $i++) {
+        $random_string .= $characters[rand(0, $character_length - 1)];
+    }
+    $object->label = $random_string . "Hat ";
```

Comment:
... and maybe get rid of the space after "hat"? or maybe that whole chunk of string altogether?